examples/2d/compute_speed.py

```diff
@@ -59,22 +85,13 @@
 
 times = 10
 
-###############################################################################
-# Determine which devices (CPU or GPU) that are supported by the current
-# backend.
-
-if backend.NAME == 'torch':
-    devices = ['cpu', 'gpu']
-elif backend.NAME == 'skcuda':
-    devices = ['gpu']
-
 ###############################################################################
 # Create the `Scattering2D` object using the given parameters and generate
 # some compatible test data with the specified batch size. The number of
 # channels in the test data here is set to `3`, corresponding to the three
 # colors channels in an RGB image.
 
-scattering = Scattering2D(J, shape=(M, N), L=L)
+
 
 x = torch.randn(batch_size, 3, M, N, dtype=torch.float32)
```

Comment:
cannot simply have torch in here without renaming file or making sure we have it